lib/beaker/dsl/install_utils.rb

```diff
@@ -478,11 +478,18 @@ def do_install hosts, opts = {}
             host['dist'] = "puppet-enterprise-#{version}-#{host['platform']}"
           elsif host['platform'] =~ /windows/
             version = host[:pe_ver] || opts['pe_ver_win']
+            should_install_64bit = !(version_is_less(version, '3.4')) and host.is_x86_64? and not host['install_32'] and not opts['install_32']
             #only install 64bit builds if
             # - we are on pe version 3.4+
             # - we do not have install_32 set on host
             # - we do not have install_32 set globally
-            if !(version_is_less(version, '3.4')) and host.is_x86_64? and not host['install_32'] and not opts['install_32']
+            if !(version_is_less(version, '4.0'))
```

Comment:
@Iristyle it is the PE version number.  It can be provided from a user in the hosts file, or it can be grabbed from the package/version file from a build.  Will that change as well?  I thought that 4.0 and "shallow gravy" were synonymous..